test/functional/master/test_api_version_requests.py

```diff
@@ -0,0 +1,25 @@
+from genty import genty, genty_dataset
+
+from test.framework.functional.base_functional_test_case import BaseFunctionalTestCase
+
+@genty
+class TestMasterAPIVersionRequests(BaseFunctionalTestCase):
+
+    def _build_accept_header_with_api_version(self, version: int):
+        header = 'Accept'
+        value = 'application/vnd.clusterrunner.v{}+json'.format(version)
+        return {header: value}
+
+    @genty_dataset(
+        no_accept_header=(None, 1),
```

Comment:
I would expect the case where no header is passed in to return the *latest* API version. Is that right?

I noticed that you didn't touch the `UrlBuilder` class which looks like it embeds the `/v1/` automatically. (That's what `master._api.url('version')` is calling into below.) Was that intentional?